manifests/service.pp

```diff
@@ -120,8 +120,7 @@
   }
 
   $_manage_service = $manage_service ? {
-    false   => undef,
-    true    => 'Service[docker]',
+    true    => Service['docker'],
     default => undef,
```

Comment:
'undef' should be '[]' (Otherwise, I get this error:

```
Error: Parameter notify failed on File[/etc/sysconfig/docker-storage-setup]: No title provided and :undef is not a valid resource reference at <xxx>docker/manifests/service.pp:135
```